internal/controllers/user_integrations_auth_controller.go

```diff
@@ -18,6 +18,8 @@ import (
 	"time"
 )
 
+const TeslaFleetAuthCacheKey = "integration_credentials_%s"
```

Comment:
Does this need to be exported?
```suggestion
const teslaFleetAuthCacheKey = "integration_credentials_%s"
```

Reply:
Yes, its being used to form the `cacheKey`

Reply:
But it's only ever being used in this package, yeah?